YOC: Initial version, implemented YOC's VIS.X bidder

jtovba opened the pull request:
## Type of Change
- [x] New adapter
- [ ] Feature Update
- [ ] Bug Fix
- [ ] Code Refactor
- [ ] Other

## Description of Change
Initial version - implementation of YOC's VIS.X bidder. In case of questions, please reach out to vladimir.miliushin@yoc.com or evgenij.tovba@yoc.com.

ix-certification commented:
Hi,

We have 3 concerns we want to highlight in file yoc-htb.js

1. In function `__parseResponse`, the code is going through every bid of every seatbid to find a match for each `returnParcel`. And the bid will be removed if a match is found. And the loop continues to go over the rest bids of the seatbid and the rest seatbids. The concern we have is that it's unnecessary to go over the same seatbid again for another `returnParcel` since one of the bids in the seatbid is already matching certain `returnParcel`. So our suggestion is to remove the seatbid if one of the bids in the seatbid is matching certain `returnParcel`.

2. Line 303, an eslint lint disable rule is added, which is unnecessary since line there's no lint error in line 304. So our suggestion is to remove line 303.

3. The version in profile is 1.0.0. Please follow our instruction in https://github.com/indexexchange/ht-wrapper-adapters#creatingAdapterModule ("version - If this is the first iteration of your module, please leave this field at 2.0.0.")

Please let us know if you have any question.

Thanks.

jtovba commented:
Hi IX team, I've updated the code in regard to your concerns no. 2 and 3.

In regard to no. 1 the loops already contain break statements, I think this should be in line with your expectations. Please check the file again, I've added more comments to hopefully explain things better.

mk0x9 commented:
@ix-certification, hi, do you have any updates on this issue?

Thanks.
